Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.yml

```diff
@@ -0,0 +1,527 @@
+category: Authentication
+commonfields:
+  id: Azure Sentinel
+  version: -1
+configuration:
```

Comment:
lets add params to set the alerts severity and limit of these to fetch

Reply:
If I understood you correctly, done - added min_severity param to filter incidents below the specified severity